src/platforms/javascript/guides/ember/ember2.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Ember 2.x
+description: "Learn how to use Sentry's Emberjs integration if you're using Ember 2.x"
+redirect_from:
+  - /clients/javascript/integrations/ember/
+---
+
+If you're using `Ember 2.x`, you can use Sentry's Ember integration.
+
+To use Sentry with your Ember application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
+
+On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
+To use ESM imports without any additional configuration, you can use `ember-auto-import`
+by installing it with `ember install ember-auto-import`.
+
+Starting with version `5.x` our `Ember` integration lives in its own `@sentry/integrations` package.
+Install the package using either `npm` or `yarn`:
+
+```bash {tabTitle:npm}
+npm install --save @sentry/integrations
+```
+
+```bash {tabTitle:yarn}
+yarn add @sentry/integrations
+```
+
+Then add to your `app.js`:
+
+```javascript
+import * as Sentry from "@sentry/browser";
+import { Ember as EmberIntegration } from "@sentry/integrations";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [new EmberIntegration()],
+});
+```
+
+If you are using the CDN version or the Loader, we provide a standalone file for every integration, to set it up [check out our CDN documentation](/platforms/javascript/configuration/install/cdn)
+
+<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
```

Comment:
This is using the JS example - which is completely fine if it's technically correct.